src/styleguide/_patterns/00-particles/00-palettes/02-yellows.json

```diff
@@ -0,0 +1,89 @@
+{
+  "colors": [
+    {
+      "color": {
+        "name": "Latte",
+        "varname": "$color-buildit-latte",
+        "hex": "4f2f00",
+        "wcag": {
+          "light": {
+            "small": "pass",
+            "big": "pass"
+          },
+          "dark": {
+            "small": "fail",
+            "big": "fail"
+          }
+        }
+      }
+    },
+    {
+      "color": {
+        "name": "Cuppa",
+        "varname": "$color-buildit-cuppa",
+        "hex": "ac6700",
+        "wcag": {
+          "light": {
+            "small": "fail",
+            "big": "pass"
+          },
+          "dark": {
+            "small": "pass",
+            "big": "pass"
+          }
+        }
+      }
+    },
+    {
+      "color": {
+        "name": "Egg Yolk",
+        "varname": "$color-buildit-egg-yolk",
+        "hex": "ff9900",
+        "wcag": {
+          "light": {
+            "small": "fail",
+            "big": "fail"
+          },
+          "dark": {
+            "small": "pass",
+            "big": "pass"
+          }
+        }
+      }
+    },
+    {
+      "color": {
+        "name": "Murican Cheese",
```

Comment:
Can we put the apostrophe back in?

**`Murican**

I think that makes the fact that it's derived from "**A**merican" clearer.